soh/src/code/padmgr.c

```diff
@@ -303,6 +303,13 @@ void PadMgr_HandleRetraceMsg(PadMgr* padMgr) {
     }
     osRecvMesg(queue, NULL, OS_MESG_BLOCK);
     osContGetReadData(padMgr->pads);
+
+    if (CVar_GetS32("gRumbleEnabled", 0) != 0) {
+        padMgr->padStatus[0].status = 1;
+    } else {
+        padMgr->padStatus[0].status = 0;
+    }
+
```

Comment:
Since status will have the same value as gRumbleEnabled, you could replace this whole if block with 

padMgr->padStatus[0].status = CVar_GetS32("gRumbleEnabled", 0);